pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -178,7 +178,7 @@ var _ = Describe("Pod Network", func() {
 		mockNetwork.EXPECT().LinkSetMaster(dummy, bridgeTest).Return(nil)
 		mockNetwork.EXPECT().AddrAdd(bridgeTest, bridgeAddr).Return(nil)
 		mockNetwork.EXPECT().StartDHCP(testNic, bridgeAddr, api.DefaultBridgeName, nil)
-		mockNetwork.EXPECT().CreateTapDevice().Return(tapDeviceName, nil)
+		mockNetwork.EXPECT().CreateTapDevice(false).Return(tapDeviceName, nil)
```

Comment:
The name should be removed from the return value. Same for all the occurrences of `CreateTapDevice' in the test.